public/stylesheets/detailsPage.css

```diff
@@ -0,0 +1,63 @@
+.category-color {
+  background: #F5DECD;
+}
+
+#category-bg {
+  position: relative;
+  width: 100%;
+  height: auto;
+  height: 200px;
+}
+
+.app-logo {
+  width: 200px;
+  height: 200px;
+  background-size: cover;
+  border-radius: 8px;
+  position: absolute;
+  bottom: -100px;
+  left: 0;
+  right: 0;
+  margin: auto;
+}
+
+.app-title {
+  margin-top: 140px;
+  text-align: center;
+}
+
+.description {
+  padding: .5rem!important; 
+}
+
+.btn {
+  color: black;
+  border-color: black;
+  box-shadow: none;
+}
```

Comment:
You will change every button in other pages if you change the .btn like this.
Please add an id (or class) to your detailsPage and use it for all of your css elector, so that it wont affect other pages. See example on header page css

Reply:
I added an id details-btn to fix this.